buildSrc/src/main/kotlin/Releases.kt

```diff
@@ -119,20 +119,32 @@ fun Project.publishArtifact(artifact: LibraryArtifact) {
             licenses {
               license {
                 name.set("The Apache License, Version 2.0")
-                url.set("http://www.apache.org/licenses/LICENSE-2.0.txt")
+                url.set("https://www.apache.org/licenses/LICENSE-2.0.txt")
               }
             }
           }
           repositories {
             maven {
               name = "CI"
-              url = uri("file://${rootProject.buildDir}/ci-repo")
+              url =
+                if (System.getenv("REPOSITORY_URL") != null) {
```

Comment:
NIT: Similarly also add for REPOSITORY_URL
```suggestion
                // REPOSITORY_URL is defined in .github/workflows/build.yml
                if (System.getenv("REPOSITORY_URL") != null) {
```